pypi_formula_mappings.json

```diff
@@ -459,22 +600,41 @@
   },
   "semgrep": {
     "package_name": "semgrep",
-    "exclude_packages": ["jsonschema", "attrs", "pyrsistent", "typing-extensions"]
+    "exclude_packages": [
+      "jsonschema",
+      "attrs",
+      "pyrsistent",
+      "typing-extensions"
+    ]
   },
   "sgr": {
-    "exclude_packages": ["typing-extensions", "jsonschema", "attrs", "pyrsistent"]
+    "exclude_packages": [
+      "typing-extensions",
+      "jsonschema",
+      "attrs",
+      "pyrsistent"
+    ]
   },
   "shallow-backup": {
     "exclude_packages": ["six"]
   },
+  "shyaml": {
+    "exclude_packages": ["PyYAML"]
+  },
   "snakemake": {
-    "exclude_packages": ["tabulate", "PyYAML", "jsonschema", "attrs", "pyrsistent"]
+    "exclude_packages": [
+      "tabulate",
+      "PyYAML",
+      "jsonschema",
+      "attrs",
+      "pyrsistent"
+    ]
   },
   "sphinx-doc": {
     "extra_packages": ["numpydoc"]
   },
   "sqlfluff": {
-    "exclude_packages": ["PyYAML"]
+    "exclude_packages": ["PyYAML", "typing-extensions"]
```

Comment:
Definitely not unchanged

Reply:
Oops, I did this after making #112865. I'll revert this change.